ui/tests/integration/components/job-status-panel-test.js

```diff
@@ -0,0 +1,142 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { find, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { startMirage } from 'nomad-ui/initializers/ember-cli-mirage';
+import { initialize as fragmentSerializerInitializer } from 'nomad-ui/initializers/fragment-serializer';
+import { componentA11yAudit } from 'nomad-ui/tests/helpers/a11y-audit';
+
+module(
+  'Integration | Component | job status panel | active deployment',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    hooks.beforeEach(function () {
+      fragmentSerializerInitializer(this.owner);
+      window.localStorage.clear();
+      this.store = this.owner.lookup('service:store');
+      this.server = startMirage();
+      this.server.create('namespace');
+    });
+
+    hooks.afterEach(function () {
+      this.server.shutdown();
+      window.localStorage.clear();
+    });
+
+    test('there is no latest deployment section when the job has no deployments', async function (assert) {
+      this.server.create('job', {
+        type: 'service',
+        noDeployments: true,
+        createAllocations: false,
+      });
+
+      await this.store.findAll('job');
+
+      this.set('job', this.store.peekAll('job').get('firstObject'));
```

Comment:
`nit`: You could simplify this to

```
const job = this.server.create('job', options);
this.set('job', job);
```

We do have this pattern in our codebase, however, I'm guessing its from 4-5 years ago and the API was different.